samcli/local/init/templates/cookiecutter-aws-sam-hello-nodejs/{{cookiecutter.project_name}}/README.md

```diff
@@ -1,211 +1,139 @@
 
-If your dependencies contain native modules that need to be compiled specifically for the operating system running on AWS Lambda, use this command to build inside a Lambda-like Docker container instead:
 ```bash
-sam build --use-container
+my-application$ sam logs -n HelloWorldFunction --stack-name {{ cookiecutter.project_name }} --tail
 ```
 
-By default, this command writes built artifacts to `.aws-sam/build` folder.
+You can find more information and examples about filtering Lambda function logs in the [SAM CLI Documentation](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-logging.html).
 
-## SAM and AWS CLI commands
+## Unit tests
 
-All commands used throughout this document
+Tests are defined in the `hello-world/tests` folder in this project. Use NPM to install the [Mocha test framework](https://mochajs.org/) and run unit tests.
 
 ```bash
-# Invoke function locally with event.json as an input
-sam local invoke HelloWorldFunction --event event.json
-
-# Run API Gateway locally
-sam local start-api
+my-application$ cd hello-world
+hello-world$ npm install
+hello-world$ npm run test
+```
 
-# Create S3 bucket
-aws s3 mb s3://BUCKET_NAME
+## Cleanup
 
-# Package Lambda function defined locally and upload to S3 as an artifact
-sam package \
-    --output-template-file packaged.yaml \
-    --s3-bucket REPLACE_THIS_WITH_YOUR_S3_BUCKET_NAME
+To delete the sample application and the bucket that you created, use the AWS CLI.
 
-# Deploy SAM template as a CloudFormation stack
-sam deploy \
-    --template-file packaged.yaml \
-    --stack-name {{ cookiecutter.project_name.lower().replace(' ', '-') }} \
-    --capabilities CAPABILITY_IAM
+```bash
+my-application$ aws cloudformation delete-stack --stack-name {{ cookiecutter.project_name }}
+my-application$ aws s3 rb s3://BUCKET_NAME
+```
 
-# Describe Output section of CloudFormation stack previously created
-aws cloudformation describe-stacks \
-    --stack-name {{ cookiecutter.project_name.lower().replace(' ', '-') }} \
-    --query 'Stacks[].Outputs[?OutputKey==`HelloWorldApi`]' \
-    --output table
+## Resources
 
-# Tail Lambda function Logs using Logical name defined in SAM Template
-sam logs -n HelloWorldFunction --stack-name {{ cookiecutter.project_name.lower().replace(' ', '-') }} --tail
-```
+See the [AWS SAM developer guide](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html) for an introduction to SAM specification, the SAM CLI, and serverless application concepts.
 
-**NOTE**: Alternatively this could be part of package.json scripts section.
+Next, you can use AWS Serverless Application Repository to deploy ready to use Apps that go beyond hello world samples and learn how authors developed their applications: [AWS Serverless Application Repository main page](https://aws.amazon.com/serverless/serverlessrepo/)
```

Comment:
Does mentioning some specific SAR apps also make sense? helps the user to continue the journey to more complex apps.